src/logic/currentSession/store/actions/addViewedSafe.ts

```diff
@@ -1,6 +1,8 @@
+import { Dispatch } from 'redux'
+
 import updateViewedSafes from 'src/logic/currentSession/store/actions/updateViewedSafes'
 
-const addViewedSafe = (safeAddress) => (dispatch) => {
+const addViewedSafe = (safeAddress: string) => (dispatch: Dispatch): void => {
```

Comment:
actually this should be of type `ThunkDispatch` from `redux-thunk` package, we've been doing it wrong :(

Reply:
Done